.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
       - '!main'
 
 env:
-  BUILDER_VERSION: v0.7.7
+  BUILDER_VERSION: v0.8.9
```

Comment:
v0.8.11
has speed improvements for CI

Reply:
I'll try updating to that version, but if I see any new/different isssue. I'm leaving in favor of something working now